uast/uastyaml/uastyaml.go

```diff
@@ -0,0 +1,345 @@
+package uastyaml
```

Comment:
This is just the renamed file, right? (I wish GitHub distinguished moves better)

Reply:
Correct. I also made temporary aliases in the old package.